db/keyval/etcd/plugin_impl_etcd.go

```diff
@@ -31,7 +31,7 @@ const (
 	// healthCheckProbeKey is a key used to probe Etcd state
 	healthCheckProbeKey = "/probe-etcd-connection"
 	// ETCD reconnect interval
-	defaultReconnectInterval = 2
+	defaultReconnectInterval time.Duration = 2
```

Comment:
This here means `defaultReconnectInterval` is `2` nanoseconds. You need to use `x * time.Second`.